portfolio/src/main/webapp/photos.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="navbar">
+        <ul>
+          <li><a href="index.html">home</a></li> 
+          <li><a href="projects.html">projects</a></li> 
+          <li><a href="blog.html">blog</a></li> 
+          <li><a href="photos.html">photos</a></li> 
+          <li><a href="random.html">random</a></li>
+        </ul>
+    </div> 
+    
+    <div id="photoscontent">
+        <div id="photosdescrip">
+            <h1>Photos + nice memories</h1>
+            <h4>Amateur photography shot on (arguably outdated) iPhone 6</h4> 
```

Comment:
btw is there a reason why this jumps from h1->h4. and then later in the html there's only h3,h4 with no h2's anywhere in the HTML.

Reply:
I initially thought of a different structure but then decided on something else, changed this to a h2